53507 ER4 Report

tyndmar opened the pull request:
Program changed:
cerep/r-prepN.w
removed "procat" (RM Category) from selectable fields

Commit:
53507 ER4 Report
Program changed:
cerep/r-prepN.w
removed "procat" (RM Category) from selectable fields

brad-advantzware reviewed (changes requested):
Spec changed to convert prep.procat to item.procat.

Commit:
53507 ER4 Report
File changed: cerep/r-prepN.w
Re-added procat field; assign procat value from RM ite.procat record
